Swat/SwatGroupedFlydown.php

```diff
@@ -1,40 +1,34 @@
 <?php
 
 /**
- * A tree flydown input control that displays flydown options in optgroups
+ * A tree flydown input control that displays flydown options in optgroups.
  *
  * The tree for a grouped flydown may be at most 3 levels deep including the
  * root node.
  *
- * @package   Swat
  * @copyright 2006-2016 silverorange
  * @license   http://www.gnu.org/copyleft/lesser.html LGPL License 2.1
  */
 class SwatGroupedFlydown extends SwatTreeFlydown
 {
-    // {{{ public function setTree()
-
     /**
-     * Sets the tree to use for display
+     * Sets the tree to use for display.
      *
      * The tree for a grouped flydown may be at most 3 levels deep including
      * the root node.
      *
-     * @param SwatDataTreeNode $tree the tree to use for display.
+     * @param SwatTreeFlydownNode $tree the tree to use for display
      *
-     * @throws SwatException if the tree more than 3 levels deep.
+     * @throws SwatException if the tree more than 3 levels deep
      */
-    public function setTree(SwatTreeFlydownNode $tree)
+    public function setTree(SwatTreeNode $tree): void
```

Comment:
Assume the class change from `SwatDataTreeNode` to `SwatTreeFlydownNode` is intentional, but flagging just in case.

Reply:
Yes, so it matches the signature of the parent `SwatTreeFlydown::setTree()` method.